opencda/core/ml_libs/rl/utils/simulator_utils/sensor_utils.py

```diff
@@ -0,0 +1,538 @@
+import os
+import copy
+import logging
+import time
+import numpy as np
+import carla
+import math
+import weakref
+import shapely.geometry
+from enum import Enum
+from easydict import EasyDict
+from collections import deque
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+from opencda.core.ml_libs.rl.simulators.carla_data_provider import CarlaDataProvider
+from ding.utils.default_helper import deep_merge_dicts
+
+DEFAULT_CAMERA_CONFIG = {
+    'size': [384, 160],
+    'fov': 90,
+    'position': [2.0, 0.0, 1.4],
+    'rotation': [0, 0, 0],
+}
+
+DEFAULT_CAMERA_AUG_CONFIG = {'position_range': [0, 0, 0], 'rotation_range': [0, 0, 0]}
+
+DEFAULT_LIDAR_CONFIG = {
```

Comment:
Better define these configurations in the yaml file. Right now the configurations are defined across several files, which may make the changing of parameters harder to maintain.